script.js

```diff
@@ -17,3 +22,29 @@ btnmenu.addEventListener('click', () => {
 btnclose.addEventListener('click', () => {
   item.style = 'display: none;';
 });
+
+const data = [];
+submit.addEventListener('click', (e) => {
+  if (emailval.value === emailval.value.toLowerCase() && emailval.value !== '') {
+    data[0] = Myname.value;
+    data[1] = emailval.value;
+    data[2] = comments.value;
+    localStorage.setItem('local', JSON.stringify(data));
+  } else {
+    terror.textContent = 'Error!! Please add valid email';
+    e.preventDefault();
+  }
+});
+
```

Comment:
- [ ] You have done great so far implementing some of the changes requested by the previous reviewer, :+1: But I think there are some changes you still need to make before moving to the next milestone. :+1: 
- [ ] As pointed out by the previous reviewer, even after filling in a valid email, the form still doesn't submit, it shows an error in the console, Take a look at the video below. I'll suggest you rectify this  :smile: :wave: 

- - Video:

https://github.com/Wils-FOURR-speed/My-portfolio/assets/107632780/79973fa2-db09-4960-a727-3467277fcf5c